src/System.Text.RegularExpressions/tests/Regex.Cache.Tests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Diagnostics;
+using Xunit;
+
+namespace System.Text.RegularExpressions.Tests
+{
+    public class Regex_Cache_Tests : RemoteExecutorTestBase
+    {
+        [Fact]
+        public void CacheSize_Get()
+        {
+            Assert.Equal(15, Regex.CacheSize);
```

Comment:
You are testing here if a property getter method correctly returns a fields value. I don't think we need this test.

Reply:
Is this just for code coverage?